ios/FluentUI/Pill Button Bar/PillButtonBar.swift

```diff
@@ -313,8 +313,14 @@ open class PillButtonBar: UIScrollView {
             buttons.append(button)
             stackView.addArrangedSubview(button)
 
-            // in case pillbuttonbar is used as .tabbar, adding our own index would be repetitive
-            if self.accessibilityTraits != .tabBar {
+            var shouldAddAccessibilityHint: Bool = true
+            if #available(iOS 14.6, *) {
+                // in case pillbuttonbar is used as .tabbar, adding our own index would be repetitive
+                // However, iOS 14.0 - 14.5 `.tabBar` accessibilityTrait does not read out the index automatically
+                shouldAddAccessibilityHint = self.accessibilityTraits != .tabBar
```

Comment:
`accessibilityTraits` is an `OptionSet` (basically a fancy Swift bitmask), so you'll want to check `contains` instead:

```swift
shouldAddAccessibilityHint = !self.accessibilityTraits.contains(.tabBar)
```

Reply:
great catch!